### PR TITLE
Awkward Goslings

### PR DESCRIPTION
**Pull request by r0ost3r**
add fancy goose sprites

DO NOT MERGE BEFORE:
* #8

### CODE REVIEW

**Comment by r0ost3r** [comment]
All emojis designed by [OpenMoji](https://openmoji.org/) – the open-source emoji and icon project. License: [CC BY-SA 4.0](https://creativecommons.org/licenses/by-sa/4.0/#)

**Comment by Necktschnagge** [comment]
https://openmoji.org/library/emoji-1F9A2/
Please always keep links to the original sources! @r0ost3r 😠

File: src/mainwindow.cpp [hunk 1]
```diff
@@ -23,6 +23,7 @@ MainWindow::MainWindow(QWidget* parent)
 	guiInteractiveController(this)
 {
 	ui->setupUi(this);
+	this->setWindowTitle("Awkward Goslings");
```
Comment (Necktschnagge) [inline src/mainwindow.cpp:26]: You should do this in *.ui (the Qt xml file.)

Comment (Necktschnagge) [reply src/mainwindow.cpp:26]: but don't do it here or now

**Comment by r0ost3r** [comment]
> https://openmoji.org/library/emoji-1F9A2/ Please always keep links to the original sources! @r0ost3r 😠

🐐 👅

https://openmoji.org/faq/
-> What is the suggested attribution for OpenMoji?

**Comment by Necktschnagge** [comment]
Obsolete, replaced by
* #53

**Pull request closed without merging**